language/move-vm/runtime/src/loader.rs

```diff
@@ -728,21 +747,29 @@ impl Loader {
             Ok(module)
         }
 
-        if let Some(module) = self.module_cache.lock().unwrap().module_at(id) {
-            return Ok(module);
+        {
+            let _timer = LIBRA_MOVEVM_LOAD_MODULE_CACHE_SECONDS.start_timer();
+            if let Some(module) = self.module_cache.lock().unwrap().module_at(id) {
+                return Ok(module);
+            }
         }
 
-        let bytes = match data_store.load_module(id) {
-            Ok(bytes) => bytes,
-            Err(err) if verify_no_missing_modules => return Err(err),
-            Err(err) => {
-                error!("[VM] Error fetching module with id {:?}", id);
-                return Err(expect_no_verification_errors(err));
+        let bytes = {
+            let _timer = LIBRA_MOVEVM_DATA_STORE_LOAD_MODULE_SECONDS.start_timer();
```

Comment:
this is another one of those API that has massive significant difference depending on whether the module is loaded or not, I am wondering if we should measure just the act of loading when we know we are loading (so deeper than here in the call stack). I am not sure how people feel about this...